moveit_ros/planning/plan_execution/src/plan_with_sensing.cpp

```diff
@@ -107,7 +106,11 @@ plan_execution::PlanWithSensing::PlanWithSensing(
       try
       {
         if (node_->get_parameter("moveit_sensor_manager", manager))
+        {
           sensor_manager_ = sensor_manager_loader_->createUniqueInstance(manager);
+          if (!sensor_manager_->initialize(node_))
+            RCLCPP_ERROR_STREAM(LOGGER, "Failed to initialize " << manager);
```

Comment:
Shouldn't the sensor manager be unset in that case?

Reply:
Nice catch, thanks!